app/controller/issues.php

```diff
@@ -149,7 +149,19 @@ public function index($f3)
 
         // Get filter
         $args = $f3->get("GET");
-        list($filter, $filter_str) = $this->_buildFilter();
+
+        // load all issues if user is admin, otherwise load by group access
+        $user = $f3->get("user_obj");
+        if ($user->role == 'admin') {
+            list($filter, $filter_str) = $this->_buildFilter();
+        } else {
+            $helper = \Helper\Dashboard::instance();
+            $groupString = implode(",", $helper->getGroupIds()) . "," . $user->id;
```

Comment:
I think this may create an invalid string if the user is not in any groups. It's probably ideal to add `$user->id` to the return value array of `$helper->getGroupIds()` here rather than concatenating. For example:

```php
$groupString = implode(",", array_merge($helper->getGroupIds(), [$user->id]));
```